### DISCUSSION TITLE
[Slideshow/Album Feature] <Sort Settings>

### DISCUSSION THREAD

**Discussion by Philshappy** [immich-app/immich#4672] [Feature Request]
### The feature

Add a slide show and Album setting feature that includes sort order (date and name) and for slide show transition time.  Currently the sort order is always from recent to latest and that makes a Non usable slide show for most things as well as difficult to view an album from most recent to oldest when showing an album of something that is a particular order such as a trip. Adding an option for a basic transition *fade in-out) would be a great future feature.

### Platform

- [X ] Server
- [x] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1689
